src/libpriv/rpmostree-origin.c

```diff
@@ -469,11 +488,19 @@ rpmostree_origin_set_rojig_description (RpmOstreeOrigin *origin,
 }
 
 gboolean
-rpmostree_origin_set_rebase (RpmOstreeOrigin *origin,
-                             const char      *new_refspec,
-                             GError         **error)
-{
-  /* We don't want to carry any commit overrides or version pinning during a
+rpmostree_origin_set_rebase_custom (RpmOstreeOrigin *origin,
+                                    const char      *new_refspec,
+                                    const char      *custom_origin_url,
+                                    const char      *custom_origin_description,
+                                    GError         **error)
+{
+  /* Require non-empty strings */
+  if (custom_origin_url)
+    g_return_val_if_fail (*custom_origin_url, FALSE);
+  if (custom_origin_description)
+    g_return_val_if_fail (*custom_origin_description, FALSE);
```

Comment:
Right now if we pass `--custom-origin-url` without `--custom-origin-description`, we'll hit this, no? Seems like we should drop this check for `custom_origin_description` at least. And/or also canonicalize empty string to `NULL` once on the daemon side.

Reply:
True.  You know, there's no good reason to make it optional, let's just require both.  Done :arrow_down: